com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/LuaBuilder.java

```diff
@@ -534,11 +534,13 @@ public void build(Task task) throws CompileExceptionError, IOException {
         // this is unacceptable for html5 games where size is a key factor
         // see https://github.com/defold/defold/issues/6891 for more info
         if (useLua51) {
+            constructLuaJITBytecode(task, script, false);
```

Comment:
Compile Lua code using LuaJit even if we are not going to use the byteode it produces